Makefile

```diff
@@ -89,7 +89,7 @@ oldtest: | nvim helptags
 	+$(SINGLE_MAKE) -C src/nvim/testdir $(MAKEOVERRIDES)
 
 helptags: | nvim
-	+$(BUILD_CMD) -C build runtime/doc/tags
```

Comment:
@fwalch it exists at https://github.com/neovim/neovim/blob/77937c4edd0e559ec66811ea20b103c329627a83/runtime/CMakeLists.txt#L81                                  
 , unless I am mistaken. Though your way is less obfuscated.